core/domain/topic_jobs_one_off.py

```diff
@@ -96,6 +98,35 @@ def reduce(key, values):
             yield (key, values)
 
 
+class InteractionsInATopicAuditOneOffJob(jobs.BaseMapReduceOneOffJobManager):
+    """An audit job to report all interaction ids used in a Topic."""
+
+    @classmethod
+    def entity_classes_to_map_over(cls):
+        return [topic_models.TopicModel]
+
+    @staticmethod
+    def map(item):
+        if not item.deleted:
+            topic = topic_fetchers.get_topic_from_model(item)
+            story_references = topic.get_all_story_references()
+            interaction_ids = set()
+
+            for story_reference in story_references:
```

Comment:
Interactions inside questions are omitted on purpose?

Reply:
Oops, no, we need it. Thanks for pointing it out! I'll add that as well.

Reply:
Okay.